### DISCUSSION TITLE
darknet do not detect objects in odc-development branch

### DISCUSSION THREAD

**Discussion by gjghks** [opendatacam/opendatacam#562] [Q&A] [answered]
first, thank you for your great works.

odc-master branch installed without docker is working well on nvidia agx board (changed mongodb version 4.0 to 4.2.22 for odc-dev branch)

however odc-dev branch installed without docker is actually not working on same board.

although both branch linked same darknet, odc-master detects objects well, but odc-dev do not detect objects.

odc-dev prints the log of loaded weights.

![image](https://user-images.githubusercontent.com/27722614/193715071-18a35a5d-af7c-4ce6-bcc9-339110e688d9.png)

how can I fix this?

**Comment by vsaw** [comment]
What exactly does "does not count mean?" No UI? Or no CSV file?

Also: The current Dev branch contains breaking changes when it comes to the config.JSON. In your screenshot I can see you are still using the old config version. Copy the config from the development branch and see if this helps. :)

**Comment by gjghks** [comment]
thank you reply!

What exactly does "does not count mean?" No UI? Or no CSV file?
=> as you can see below figure, I use Dev branch config file and well loaded pretrained yolo weights.
however, Dev branch can not detect object.
![image](https://user-images.githubusercontent.com/27722614/193785361-be67aad4-78f5-4e1c-a2cf-2202f6f485bc.png)

Master branch can detect very well same device and darknet .

![image](https://user-images.githubusercontent.com/27722614/193792710-c0293f20-edb7-4aca-ac9a-c4ddffb16804.png)

**Comment by vsaw** [comment]
Can you check on Port 8070 if there are detections? If not, then it's a Darknet issue. 

If there are detections please post a copy of your config.json here.

**Comment by gjghks** [comment, marked as answer]
thank you reply,

I took a look your all commit on dev branch and found the point.

It took long time,, because I took a look timeline follow,, OMG,,

Commits on Jun 5, 2022
changes the track counted classes in config.json
![image](https://user-images.githubusercontent.com/27722614/195494724-98b32fdf-03cc-49fd-be4e-65893eb79af5.png)

It's only shows VALID_CLASSES.

I think this should be included in the readme.

**Reply by vsaw** [reply]
That's a good point. If you have the time I'd appreciate a PR to update the documentation 👍

**Reply by vsaw** [reply]
Here's an Issue to keep track of this https://github.com/opendatacam/opendatacam/issues/564
